### PR TITLE
Add systemd units files and instructions for docker cleaning

### PR DESCRIPTION
**Pull request by Bidski**

### CODE REVIEW

**Review by KipHamiltons — COMMENTED** [review]
I think we should add stuff to set this up to the NUbook guide to set up a buildkite agent, and to the script which does that automatically

File: doc/BuildKiteDockerClean/docker_clean.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```
Comment (KipHamiltons) [inline doc/BuildKiteDockerClean/docker_clean.sh:1]: this also presupposes bash

**Review by KipHamiltons — CHANGES_REQUESTED** [review]

**Comment by KipHamiltons** [comment]
This is _kind of_ redundant now we have GHA.. Not sure if it will be useful if and when we use self-hosted runners though. Keen to hear thoughts @Bidski @Y0sh1dk

**Comment by CMurtagh-LGTM** [comment]
We have moved to github actions, I'm assuming this in now irrelevant.

**Pull request closed without merging**